python/scannerpy/database.py

```diff
@@ -519,24 +524,36 @@ def _toposort(self, op):
 
         return [e.to_proto(eval_index) for e in eval_sorted]
 
+    def _get_output_columns(self, op_name):
+        output_columns_args = self.protobufs.OutputColumnsArgs()
```

Comment:
I would condense `GetOutputColumns` and `HasOp` into a single function that returns the information for an `Op` if it exists, and false otherwise.